src/controllers/Auth.js

```diff
@@ -0,0 +1,29 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import { User } from '../database/models';
+
+dotenv.config();
+
+class Auth {
+  static async login(req, res) {
+    const { email, password, companyId } = req.body;
+
+    const user = await User.findOne({ where: { email, companyId } });
+    if (!user) {
+      res.status(404).json({ status: 404, msg: 'incorrect email or password or company code', success: false }).end();
+    } else {
+      const hash = user.password;
+      const result = await bcrypt.compareSync(password, hash);
+      if (result === true) {
+        const payload = user;
```

Comment:
I kind of think the payload is too big, we can select just properties like userid, name and companyId to save in the token